### ISSUE TITLE
URL error for loading ari package dependency

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/leanbuild#25]
This error happens ocassionally and it doesn't have to do with this repository: 
```
The downloaded binary packages are in
	C:\Users\runneradmin\AppData\Local\Temp\Rtmp4Giw5L\downloaded_packages
Error in utils::download.file(url, path, method = method, quiet = quiet,  : 
  cannot open URL 'https://api.github.com/repos/jhudsl/ari/contents/DESCRIPTION?ref=HEAD'
Calls: saveRDS ... github_DESCRIPTION -> download -> base_download -> base_download_headers
Execution halted
Error: Process completed with exit code 1.
```

_Originally posted by @cansavvy in https://github.com/jhudsl/leanbuild/issues/24#issuecomment-908523648_

**Comment by cansavvy** [comment]
I think when I work on #32 this will be handy to reassess along with it.

**Comment by cansavvy** [comment]
This should be addressed upon #33 's merging
